src/App.js

```diff
@@ -1,24 +1,102 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
+import "antd/dist/antd.css";
+import "./index.css";
+import { Layout, Menu, Button, Row, Col, Space, Calendar } from "antd";
+import Event from "./Components/Event";
+import {
+  MenuUnfoldOutlined,
+  MenuFoldOutlined,
+  UserOutlined,
+  VideoCameraOutlined,
+  UploadOutlined,
+} from "@ant-design/icons";
+
+const { Header, Sider, Content, Footer } = Layout;
 
 function App() {
+  const [collapsed, setCollapsed] = useState(true);
+
+  const toggle = () => setCollapsed((collapsed) => !collapsed);
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Layout>
+        <Sider trigger={null} collapsible collapsed={collapsed}>
+          <div className="logo" />
+          <Menu theme="dark" mode="inline" defaultSelectedKeys={["1"]}>
+            <Menu.Item key="1" icon={<VideoCameraOutlined />}>
+              <span>Event</span>
+              <Link to="/event" />
+            </Menu.Item>
+            <Menu.Item key="2" icon={<UserOutlined />}>
+              <span>People</span>
+              <Link to="/people" />
+            </Menu.Item>
+            <Menu.Item key="3" icon={<UploadOutlined />}>
+              <span>Profile</span>
+              <Link to="/profile" />
+            </Menu.Item>
+          </Menu>
+        </Sider>
+        <Layout className="site-layout">
+          <Header className="site-layout-background" style={{ padding: 0 }}>
+            <Row align="space-between">
+              <Col>
+                {React.createElement(
```

Comment:
Any reason we need to use `createElement` here instead of `<MenuUnfoldOutlined />` and `<MenuFoldOutlined />`?

Reply:
I just copied ant design demo code here, if I do in <MenuUnfolodOutline />, it will have more code. className="trigger" onClick={toggle}  have to be used twice
{collapsed ? (
                  <MenuUnfoldOutlined className="trigger" onClick={toggle} />
                ) : (
                  <MenuFoldOutlined className="trigger" onClick={toggle} />
                )}